pkg/kubelet/volumemanager/reconciler/reconciler.go

```diff
@@ -485,9 +485,8 @@ func (rc *reconciler) reconstructVolume(volume podVolume) (*reconstructedVolume,
 		if err != nil {
 			return nil, err
 		}
-	} else {
-		uniqueVolumeName = util.GetUniqueVolumeNameForNonAttachableVolume(volume.podName, plugin, volumeSpec)
 	}
+	uniqueVolumeName = util.GetUniqueVolumeNameForNonAttachableVolume(volume.podName, plugin, volumeSpec)
```

Comment:
`else` can't be removed?
If deleted, no matter whether attachablePlugin is `nil` or not, `util.GetUniqueVolumeNameForNonAttachableVolume` be always be called.